File: back-spring-boot/src/main/java/com/groupe2/backspringboot/model/Commande.java
package com.groupe2.backspringboot.model;

import java.io.Serializable;
import java.util.List;

import javax.persistence.CollectionTable;
import javax.persistence.ElementCollection;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Version;

import lombok.Data;

@Data
@Entity
public class Commande implements Serializable{
	
	
	private static final long serialVersionUID = 7999707544138948542L;

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	
	private int idClient;
	
	@ElementCollection
	@CollectionTable(name = "liste_prestations")
	private List<String> prestations;
	
	private String date;
	
	private double total;
	
	@Version
	private int version;

}

File: front-angular/src/app/commande/commande.ts
import { LignePresta } from "../prestation/ligne-presta";

export class Commande {

    date : string;
    idClient : number;
    prestations : Array<string>;
    total : number;
 
    constructor(){
        this.date = "";
        this.idClient = 0;
        this.prestations = new Array<string>;
        this.total = 0;
    }

}
